cmd/openfga/main_test.go

```diff
@@ -569,7 +569,130 @@ func connect(t *testing.T, tester OpenFGATester) *grpc.ClientConn {
 }
 
 func GRPCWriteTest(t *testing.T, tester OpenFGATester) {
+	type output struct {
+		errorCode    codes.Code
+		errorMessage string
```

Comment:
The error message really isn't part of the API contract. I'd encourage us to functionally test just the public API contract.

Reply:
Understood that error messages aren't strictly part of the contract but our API should deliver readable, correct error messages, no? Changes in messages probably should be caught too. Definitely a gray area, but just want to ensure that users don't lose ability to resolve errors. 

Granted, I'm coming in very green so maybe the above aren't quite concerns. Happy to remove if you feel strongly.

Reply:
Going to resolve this so we can get this merged in but still interested in hearing your takes on this and if necessary will remove with a follow-up.